examples/megaclc/megaclc.cpp

```diff
@@ -3239,6 +3240,45 @@ void exec_getmegaachievements(ac::ACState& s)
     g_megaApi->getMegaAchievements(listener);
 }
 
+void exec_setCameraUploadsFolder(ac::ACState& s)
+{
+    std::unique_ptr<m::MegaNode> srcnode(g_megaApi->getNodeByPath(s.words[1].s.c_str()));
+    
+    if (!srcnode)
+    {
+        cout << "folder not found";
+    }
+    else
+    {
+        g_megaApi->setCameraUploadsFolder(srcnode->getHandle(), new OneShotRequestListener([](m::MegaApi*, m::MegaRequest *, m::MegaError* e)
+        {
+            check_err("setCameraUploadsFolder", e, ReportResult);
+        }));
+    }
+
+}
+
+void exec_getCameraUploadsFolder(ac::ACState& s)
+{
+    g_megaApi->getCameraUploadsFolder(new OneShotRequestListener([](m::MegaApi*, m::MegaRequest *r, m::MegaError* e)
+    {
+        if (check_err("getCameraUploadsFolder", e, ReportFailure))
+        {
+            unique_ptr<m::MegaNode> node(g_megaApi->getNodeByHandle(r->getNodeHandle()));
+            if (!node)
+            {
+                conlock(cout) << "No node found by looking up handle: '" << r->getNodeHandle() << "'" << endl;
```

Comment:
Perhaps it's more useful to print handles in `B64`.

Reply:
Yes, quite right.  I've added a function to make that convenient, and made the fix. thanks!